Source/Application/ITMO.Dev.ASAP.Application.Contracts/Study/Queues/Queries/GetSubmmissionsQueueBySubjectCourseGroupIds.cs

```diff
@@ -0,0 +1,11 @@
+using ITMO.Dev.ASAP.Application.Dto.Tables;
+using MediatR;
+
+namespace ITMO.Dev.ASAP.Application.Contracts.Study.Queues.Queries;
+
+internal static class GetSubmmissionsQueueBySubjectCourseGroupIds
```

Comment:
пока у нас нет кастомных очередей, так что можно обойтись таким неймингом 

```suggestion
internal static class GetSubmmissionsQueue
```